### PR TITLE
Bookmark

### PR DESCRIPTION
**Pull request by SarahWett**
Still missing functionality: 
JavaScript Coed doesn't work out switching unfilled - filled svgs.
Please review & you're welcome to share your ideas how to fix this issue! :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,51 +1,148 @@
...
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="/Users/sw/Documents/Coding/DevelopmentNF/quiz-app/components/styles.css"
```
Comment (Kheirah) [inline index.html:9]: Local paths don't work on machines other than your own😄
Use relative or absolute paths in your project. The following should work:
```js
href="./components/styles.css"
```

File: index.js [hunk 2]
```diff
@@ -1,4 +1,78 @@
...
+// --TAGLIST
+const tagList1 = document.querySelector('[data-js="tag_list1"]');
+const tagList2 = document.querySelector('[data-js="tag_list2"]');
+const tagList3 = document.querySelector('[data-js="tag_list3"]');
+// --BOOKMARK
+const bookmark1 = document.querySelector('[data-js="bookmark1"]');
+const bookmark2 = document.querySelector('[data-js="bookmark2"]');
+const bookmark3 = document.querySelector('[data-js="bookmark3"]');
+const bookmarkUnfilled = document.querySelector('[data-js="book_unfilled]');
+const bookmarkFilled = document.querySelector('[data-js="book_filled]');
```
Comment (Kheirah) [inline index.js:21]:
```suggestion
const bookmarkUnfilled = document.querySelector('[data-js="book_unfilled"]');
const bookmarkFilled = document.querySelector('[data-js="book_filled"]');
```
You are missing double quotes.

File: index.js [hunk 3]
```diff
@@ -1,4 +1,78 @@
...
 const questionCard = document.querySelector(".q-card");
+const card = document.querySelector(".card");
+// --TAGLIST
+const tagList1 = document.querySelector('[data-js="tag_list1"]');
+const tagList2 = document.querySelector('[data-js="tag_list2"]');
+const tagList3 = document.querySelector('[data-js="tag_list3"]');
+// --BOOKMARK
+const bookmark1 = document.querySelector('[data-js="bookmark1"]');
+const bookmark2 = document.querySelector('[data-js="bookmark2"]');
+const bookmark3 = document.querySelector('[data-js="bookmark3"]');
```
Comment (Kheirah) [inline index.js:19]: I think what you want is this:
```suggestion
const bookmark1 = document.querySelector('[data-js="btn1"]');
const bookmark2 = document.querySelector('[data-js="btn2"]');
const bookmark3 = document.querySelector('[data-js="btn3"]');
```

File: index.js (hunk 2 above)
Comment (Kheirah) [inline index.js:21]: Apart from the missing double quotes, you need to give your filled and unfilled bookmarks unique names.
```suggestion
const bookmark1Unfilled = document.querySelector('[data-js="book1_unfilled"]');
const bookmark1Filled = document.querySelector('[data-js="book1_filled"]');
```

File: index.js [hunk 4]
```diff
@@ -1,4 +1,78 @@
...
+const bookmarkUnfilled = document.querySelector('[data-js="book_unfilled]');
+const bookmarkFilled = document.querySelector('[data-js="book_filled]');
 
+// Functionality Bookmarks
+// Bookmark1
+bookmark1.addEventListener("click", () => {
+  bookmarkUnfilled.classList.toggle("off");
+  bookmarkFilled.classList.toggle("off");
```
Comment (Kheirah) [inline index.js:27]: And then you need to use that unique name here and in the other functions.
```suggestion
  bookmark1Unfilled.classList.toggle("off");
  bookmark1Filled.classList.toggle("off");
```

**Review by Kheirah — CHANGES_REQUESTED** [review]
Good start! Have a look the comments and try adjusting your code to make it work :)

File: index.js (hunk 2 above)
Comment (SarahWett) [reply index.js]: oh yeah you're right - that seems to be the point! Missed that 'cause I thought a unique button would work the same way but that was surely odd! Thank you very much! :)

**Pull request closed without merging**